config/sql/se/EbookSources.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE `EbookSources` (
   `EbookId` int(10) unsigned NOT NULL,
-  `Type` tinyint(4) unsigned NOT NULL,
+  `Type` enum('pg', 'pg_australia', 'pg_canada', 'ia', 'hathi_trust', 'wikisource', 'google_books', 'faded_page', 'other') DEFAULT 'other',
```

Comment:
We should just spell out each one fully, both for consistency and for clarity for future maintainers

Reply:
Sure, done.